alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxo.hs

```diff
@@ -252,7 +251,7 @@ utxoTransition ::
     ValidateScript era,
     -- instructions for calling UTXOS from AlonzoUTXO
     Embed (Core.EraRule "UTXOS" era) (AlonzoUTXO era),
-    Environment (Core.EraRule "UTXOS" era) ~ Shelley.UtxoEnv era,
+    Environment (Core.EraRule "UTXOS" era) ~ UtxoEnv era,
```

Comment:
One thing we can do here is simply bind the two type families together:

`Environment (Core.EraRule "UTXOS" era) ~ Environment (Core.EraRule "UTXO" era)`

Reply:
oh, interesting! thanks!